feegrant/index.md

```diff
@@ -0,0 +1,239 @@
+---
+parent:
+  title: Understanding the Fee Grant Module
+order: 0
+description: Use the Cosmos SDK feegrant module to grant the use of tokens to pay for fees from one account (the granter) to another account (the grantee).
+---
+
+# Feegrant Module
+
+The [feegrant](https://docs.cosmos.network/v0.44/modules/feegrant/) enables a user / contract / module (the granter) to pay the fees for someone else (the grantee) when they want to broadcast a transaction on the blockchain. A key feature of feegrant is that the granter retains full access to their tokens, and is able to revoke the allowance at any time.
+
+## Use Feegrant to Grant an Allowance
+
+An often discussed use case for the feegrant module is that blockchains or smart contracts can allow new users to start interacting with the chain or contract without having to acquire tokens first, greatly improving the onboarding experience.
+
+There are two [fee allowance types](https://docs.cosmos.network/v0.44/modules/feegrant/01_concepts.html#fee-allowance-types) that the feegrant module implements: `BasicAllowance` and `PeriodicAllowance`, the latter allows you to setup a periodic allowance to another user. In this tutorial, you will set up two tokens in your blockchain: one default token called `stake`, which will be used for fees, and another called `kudos`, for sending to your friends.
+
+You will learn how to spin up a single node network using the simulation application in Cosmos SDK (`simapp`) and set up Alice to be a validator. Then you'll have Bob send `kudos` tokens to Carol, while having zero `stake` to pay for fees. The latter will be possible because you'll let Carol grant a `BasicAllowance` to Bob.
+
+## Requirements
+
+Before you start the tutorial, you need to install the `simd` binary.
+
+Clone the `cosmos-sdk` repository:
+
+```bash
+git clone https://github.com/cosmos/cosmos-sdk
+```
+
+Change directories and check out `v0.44.0`:
+
+```bash
+cd cosmos-sdk && git checkout v0.44.0
+```
+
+Install the `simd` binary:
+
+```bash
+make install
+```
+
+Check to make sure the installation was successful:
+
+```bash
+simd version
+```
+
+You should see `0.44.0` printed to the console.
```

Comment:
```suggestion
The version output `0.44.0` is output to the console.
```

Reply:
let's not use "should" see word list https://developers.google.com/style/word-list?hl=en#letter-s 

> should, should be
> Generally avoid.
> Because should is ambiguous by definition, it can be problematic. For example, if you're telling the reader what to do, should implies that the action is recommended but optional, leaving the reader unsure about what to do.
> Clarify what you mean.

Reply:
Thanks. How about this?

```suggestion
The version number `0.44.0` is output to the console.
```